Update ru_ru.json + russian translation of README and HOW_TO_PLAY

LingVarr opened the pull request:
I missed the en_us.json update and late to the beta..... but anyway here is the update to ru_ru.json.
Also fixed a HUGE typo in the HOW_TO_PLAY.

Sollace commented:
Ah, wonderful. Thanks!

I'll give you a chance to do the most recent change before I merge this one.

LingVarr commented:
Updated ru_ru.json.
Added russian translation of README and HOW_TO_PLAY, also added badges redirecting to other languages (you might want to change them to make them look 20% cooler).

Sollace commented:
> Updated ru_ru.json. Added russian translation of README and HOW_TO_PLAY, also added badges redirecting to other languages (you might want to change them to make them look 20% cooler).

I'll give them a look. Thanks!
